### ISSUE TITLE
Missing: date added

### ISSUE THREAD

**Issue by joonaspaakko** [joonaspaakko/audible-library-extractor#1]
> At some point in summer of 2020 Audible renewed the library on the website (and mobile app, but that's irrelevant). Not only did they change the layout a little, they made content changes too. One of those changes was that they removed the `date added` = when the book was added to your library. Totally gone from the library pages. You can still sort by `date added`, you just can't see the dates. So my fallback for a while was to get it from the store page where it was still available, but only for one or maybe two months until they took that away as well. The possible workaround would be to also scrape the purchase history page and basically connect the dots, but I don't think it's worth it. 
>
> So as the data is fetched in the order they were added to your library, you can still sort based on `date added` in the library extractor's gallery.

**TL;DR The extension can't fetch `date added`.** 

- - - - 

Image of the old and new library layouts:
![image](https://user-images.githubusercontent.com/1164476/79955990-32beb880-8488-11ea-807a-ef342ca9513c.png)

**Comment by joonaspaakko** [comment]
Changelog: "Now extracts the purchase date (added date) from purchase history... Comes with a matching filter and sorter in the gallery."

- b8a87f89 Extracting added date (purchase date)
- 09e61ed0 Rendering purchase dates in the gallery and spreadsheet data
